qiskit_qec/operators/base_xp_pauli.py

```diff
@@ -104,6 +108,8 @@ def __init__(
             matrix = nmatrix
 
         self.matrix = matrix
+        self.precision = precision
+        # TODO should _num_paulis be renamed to _num_xppaulis in this and derived classes?
```

Comment:
As we are separating the Pauli and XPPauli classes I would say it is better to change the name as you suggest.